src/KeyUnit.cpp

```diff
@@ -298,7 +347,88 @@ QString KeyUnit::getKeyName( int keyCode, int modifierCode )
     else return QString("undefined key");
 }
 
+void KeyUnit::initStats()
+{
+    statsKeyTotal = 0;
+    statsTempKeys = 0;
+    statsActiveKeys = 0;
+    statsActiveKeysMax = 0;
+    statsActiveKeysMin = 0;
+    statsActiveKeysAverage = 0;
+    statsTempKeysCreated = 0;
+    statsTempKeysKilled = 0;
+}
+
+void KeyUnit::_assembleReport( TKey * pChild )
+{
+    list<TKey*> * childrenList = pChild->mpMyChildrenList;
+    for( auto it2 = childrenList->begin(); it2 != childrenList->end(); it2++)
+    {
+        TKey * pT = *it2;
+        _assembleReport( pT );
+        if( pT->isActive() ) statsActiveKeys++;
+        if( pT->isTempKey() ) statsTempKeys++;
+        statsKeyTotal++;
+    }
+}
 
+QString KeyUnit::assembleReport()
```

Comment:
I don't think it's intentional - would like to see aliases as well - but let's keep PR's scope focused.